base/applications/taskmgr/lang/it-IT.rc

```diff
@@ -343,21 +343,21 @@ BEGIN
     ID_WINDOWS_CASCADE "Sovrappone diagonalmente le finestre sul desktop"
     ID_WINDOWS_BRINGTOFRONT "Porta la finestra in vista ma senza attivarla"
     ID_HELP_TOPICS "Visualizza gli aiuti di Task Manager"
-    ID_HELP_ABOUT "Visualizza le informazioni sul program, version e copyright"
-    ID_FILE_EXIT "Esce da Task Manager"
-    ID_OPTIONS_SHOW16BITTASKS "Mostra i task a 16-bit con ntvdm.exe associato"
+    ID_HELP_ABOUT "Visualizza le informazioni sul programma, versione e copyright"
+    ID_FILE_EXIT "Esce da Gestione attività"
```

Comment:
You changed that string back to "Task Manager" in several places, but you forgot to do the same here?